mappings/net/minecraft/data/DataGenerator.mapping

```diff
@@ -3,7 +3,17 @@ CLASS net/minecraft/class_2403 net/minecraft/data/DataGenerator
 	FIELD field_11273 providers Ljava/util/List;
 	FIELD field_11274 output Ljava/nio/file/Path;
 	FIELD field_11275 LOGGER Lorg/slf4j/Logger;
+	FIELD field_38909 runningProviders Ljava/util/List;
+	FIELD field_38910 gameVersion Lnet/minecraft/class_6489;
+	FIELD field_38911 skipGeneratorRerunWarning Z
```

Comment:
This is more than skipping warning. If set to `true`, it runs data providers even if they were already run. Maybe `ignoreCache`?

Reply:
```suggestion
	FIELD field_38911 ignoreCache Z
```